.github/workflows/rebase.yml

```diff
@@ -17,6 +17,15 @@ jobs:
     if: github.event.issue.pull_request != '' && github.event.comment.body == 'GitHub, rebase'
     runs-on: ubuntu-latest
     steps:
+      - uses: khan/pull-request-comment-trigger@master
+        name: React with rocket on run
+        with:
+          trigger: 'GitHub, rebase'
+          reaction: rocket
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+      - run: ' '
```

Comment:
Without run, it doesn't work so just had put a space here.

Reply:
Please add comment above this line to explain it in code

Reply:
Solved